src/main/java/lesson02/PrimeChecker.java

```diff
@@ -0,0 +1,41 @@
+package lesson02;
+/*
+Task 5 (PrimeChecker)
+Определить, является ли число простым. Простое число - это натуральное число (больше нуля), которое делится нацело
+только на себя и на 1.
+Ограничения: n > 0.
+boolean isPrime(int n)
+Входные аргументы: int n
+Тип результата: boolean
+ */
+public class PrimeChecker {
+    public static void main (String[] args) {
+        if (isPrimeNumberChecking(ConsoleInput.inputNumberFromConsole("Please input " +
+                "integer number that greater than 0: "))){
+            System.out.println("prime");
+        } else {
+            System.out.println("not prime");
+        }
+    }
+
+    private static boolean isPrimeNumberChecking (int n){
```

Comment:
Which extra meaning added by `Checking` in the function name?

Reply:
This was used to avoid method and boolean variable naming duplication. Since variable was removed - method renamed